app/editor/src/components/form/checkbox/styled/Checkbox.tsx

```diff
@@ -19,6 +19,8 @@ export const Checkbox = styled.div<ICheckboxProps>`
     }};
   }
 
+  margin-bottom: 1%;
```

Comment:
Anytime I see percentage I worry.  I don't know the best way, but presently there is all kind of layout issues with elements not exactly in the same vertical or horizontal line.